Sources/Public/LottieConfiguration.swift

```diff
@@ -91,5 +98,16 @@ extension RenderingEngineOption: RawRepresentable, CustomStringConvertible {
   public var description: String {
     rawValue
   }
+}
+
+// MARK: - DecodingStrategy
+
+/// How animation files should be decoded
+public enum DecodingStrategy: Hashable {
+  /// Use Codable. This is the default strategy introduced on Lottie 3.
+  case codable
 
+  /// Manually deserialize a dictionary into an Animation. This should be faster,
+  /// but since it's manually implemented, there might be issues while it's experimental.
```

Comment:
It would be nice to include discussion here about how much faster the `dictionaryBased` implementation is than the `codable` implementation. Also, if we do [make it the default](https://github.com/airbnb/lottie-ios/pull/1561/files#r866326477), we probably don't want to call it experimental here.